ci_framework/roles/os_must_gather/tasks/build_openstack-must-gather_image.yml

```diff
@@ -0,0 +1,51 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Get git tag for image tagging
+  register: openstack_must_gather_tag
+  ansible.builtin.command:
+    cmd: git show-ref --head --hash head # noqa: command-instead-of-module
+    chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/openstack-must-gather"
```

Comment:
We can also add one more ansible var to store the must-gather repo path. It will be useful for local development having different local source path.